tools/hifiasm/hifiasm.xml

```diff
@@ -268,20 +268,44 @@
                 </param>
             </when>
         </conditional>
-        <param name="log_out" type="boolean" label="Output log file?" truevalue="yes" falsevalue="no"/> 
+        <conditional name="bins">
+            <param name="bin_selector" type="select" label="Do you want the .bin files?">
+                <option value="no">no (default)</option>
+                <option value="yes">yes</option>
+            </param>
+            <when value="no" />
+            <when value="yes" />
+        </conditional>
+        <param name="log_out" type="boolean" label="Output log file?" truevalue="yes" falsevalue="no" /> 
     </inputs>
     <outputs>
+        <!-- Yes to bin files -->
+        <data name="ec_bin" format="binary" from_work_dir="output.ec.bin" label="${tool.name} on ${on_string}: ec .bin file">
```

Comment:
Would a collection make more sense in this case?

Reply:
hmm initially i had it as a boolean plus collection, but switched because 1) thought they might be easier to export as individual items, and 2) i was having trouble getting the wrapper to not create an empty collection in the case of the boolean being false... 😓

Reply:
Mh, you can download entire collections and you can dig into them to download single files. Is that not good enough?

Reply:
I can help with the empty-collection hopefully :)

Reply:
oh i forgot about downloading the whole collection, that actually would be handy for these haha -- ok i'll change it! pls take a look at the empty collection when you have a chance, thank you 😄